src/handlers/parseRawDependencyDataHandler.ts

```diff
@@ -66,10 +68,13 @@ function parseTreeNodes(treecontent: string, eol: string, indent: string, prefix
         lines.forEach(line => {
             curIndentCnt = line.indexOf(prefix);
             curNode = toDependency(line);
+            let uri: vscode.Uri;
+            let curFilePath: string;
             if (curIndentCnt === 0) {
                 curNode.root = curNode;
                 rootNode = curNode;
                 parentNode = curNode;
+                curFilePath = `${curNode.groupId}.${curNode.artifactId}`;
```

Comment:
Can it be `groupid/artifactid`? That would make more sense. Or is there any pitfall?

Reply:
since now `propagate = false`,  `groupid/artifactid` is ok. Otherwise, the same `groupid` may import errors.